lib/iris/tests/unit/tests/stock/test_netcdf.py

```diff
@@ -0,0 +1,141 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""Unit tests for the :mod:`iris.tests.stock.netcdf` module."""
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests
+
+import shutil
+import tempfile
+
+from iris import load_cube
+from iris.tests.stock import netcdf
+from iris.experimental.ugrid import MeshCoord, PARSE_UGRID_ON_LOAD
+
+
+class XIOSFileMixin(tests.IrisTest):
+    @classmethod
+    def setUpClass(cls):
+        # Create a temp directory for transient test files.
+        cls.temp_dir = tempfile.mkdtemp()
+
+    @classmethod
+    def tearDownClass(cls):
+        # Destroy the temp directory.
+        shutil.rmtree(cls.temp_dir)
+
+    def create_synthetic_file(self, **create_kwargs):
+        # Should be overridden to invoke one of the create_file_ functions.
+        # E.g.
+        # return create_file__xios_2d_face_half_levels(
```

Comment:
Just for consistency..
```suggestion
        # return netcdf.create_file__xios_2d_face_half_levels(
```

Reply:
72538e7bd8586676944df8272693937b374be9a9